### PR TITLE
Set up basic app

### PR DESCRIPTION
**Pull request by vincerubinetti**
- sets up basic app with Vue CLI with the following options
    - yarn (instead of npm)
    - vue 3
    - typescript
    - eslint and prettier
    - vuex (global state manager)
    - vue router
    - jest (unit testing)
    - cypress (e2e testing)
    - sass (with dart-sass)
- make and setup favicons with realfavicongenerator.net
- add metadata in index.html with metatags.io and others
- add jest-axe (for automated accessibility testing, alongside unit and e2e)
- add a11y eslint rules
- add ability to import yaml
- add webpack plugin to generate sitemap
- add markdown renderer
- add ability to import raw text and markdown
- add fontawesome
- add github actions workflow to deploy live to gh pages, and to run tests on pull request. split tests into separate jobs for separate checkmarks, and share yarn install cache
- make some basic tests
- setup netlify preview
- add review and check requirements before merge

Tried and not going to pursue further:
- yarn 2 (although over a year old, this change is so breaking that it's still too new and dangerous to try, and not supported officially by vue cli)

### CODE REVIEW

File: .browserslistrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+> 1%
```
Comment (kshefchek) [inline .browserslistrc:1]: I can't recall the exact scenario, but we ended up making this more liberal in our current app, >.10% and removed the last 2 versions constraint. Curious your thoughts on this

File: README.md [hunk 2]
```diff
@@ -1 +1,19 @@
 <p align="center"><img src="https://user-images.githubusercontent.com/8326331/133301616-504b30a2-a015-4de8-b611-60539f133cfc.png" height="200px" /></p>
+
+## Development
+
+### Setup
+
+Install [Node](https://nodejs.org/) and [Yarn](https://yarnpkg.com/) if you haven't already.
```
Comment (kshefchek) [inline README.md:7]: Do we want to add anything about NVM and node versions?

Comment (falquaddoomi) [reply README.md:7]: It looks like GH actions is using node 14, which is the latest stable release, so we might put something in about using at least v14.x to match our build process. NVM seems a bit outside the scope of this README to me, but if you both think it's helpful I don't think it'd hurt.

**Review by falquaddoomi — APPROVED** [review]
I was able to get the site up and running locally without any issues, and I didn't see anything wrong with the code (admittedly I only glanced over most of the boilerplate bits), so I approve.

File: README.md (hunk 2 above)
Comment (vincerubinetti) [reply README.md:7]: I normally wouldn't include this, but I'll add a note.

Comment (kshefchek) [reply README.md:7]: I'm out of the loop on node and versioning, so if you think it's not needed I trust your judgement

File: .browserslistrc (hunk 1 above)
Comment (kshefchek) [reply .browserslistrc:1]: i wish I had taken better notes on what exactly caused this, I think it was related to me not updating firefox for a few months and a new function not being supported. I know we had it set to 1% for a year or so it was fine, so we could always take the more strict approach until something breaks.

Comment (vincerubinetti) [reply .browserslistrc]: As part of habit, I won't be using any javascript or css features that aren't supported by all modern browsers (everything but IE). So this shouldn't be very relevant.

Also re: `last 2 versions`, I want to keep an attitude of forward-looking/modern, and also avoid situations where a really old version still has support, like in the case of IE9 having `0.12%` for some unholy reason.

**Pull request merged**